calico-enterprise/operations/monitor/metrics/recommended-metrics.mdx

```diff
@@ -0,0 +1,574 @@
+| Threshold breach symptoms        | Counters rising when there are no Felix or Typha restarts, or no action that could cause restarts (an upgrade for example), could indicate unexpected Felix or Typha restarts or issues. |
+| Threshold breach recommendations | Check resource usage on Typha(s) and Kubernetes nodes. Increase resources if needed. |
+| Priority level                   | Optional.                                                    |
+
+### Client connections actively streaming
+
+| Client connections actively streaming |                                                              |
+| ------------------------------------- | ------------------------------------------------------------ |
+| Metric                                | <code>sum by (instance) (typha_connections_streaming)</code> |
+| Example value                         | <code>{instance="10.0.1.20:9093"} 10</code> <br/><code>{instance="10.0.1.31:9093"} 5</code> |
+| Explanation                           | Current number of active connections that are "streaming" (have completed the handshake), to this Typha. After a connection has been Accepted (reported in the previous metric), there will be a handshake before the connection is deemed to be actively streaming. This indicates how many clients are connected to a Typha. The sum reflects per-cache metrics as well. |
+| Threshold value recommendation        | Compare the value for **Total Connections Accepted** and **Client Connections Actively Streaming**. The fluctuation of these values should be in-sync with each other if Accepted Connections are turning into Actively Streamed connections. If there is a discrepancy , you should investigate. **Note**: As always, it is recommended to baseline the relationship between these two metrics to have a sense of what is normal. It is also worth noting that in smaller clusters, it is normal for Typha to be unbalanced. Typha can handle hundreds of connections so it is of no concern if all nodes in a 10-node cluster (for example) connect to the same Typha. |
+| Threshold breach symptoms             | Felix is not getting updates from Typha. {{prodname}} network policies are out-of-sync. |
+| Threshold breach recommendations      | Check Typha and Felix logs, and rollout restart Typha(s) if needed. |
+| Priority level                        | Recommended.                                                 |
+
+### Rebalanced client connections
+
+| Rebalanced client connections    |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | <code>rate(typha_connections_dropped{$_rate_interval})</code> |
```

Comment:
I don't think rate_interval is explained anywhere (and elsewhere we use `30s`)?  Will users know what this is?

Reply:
What is the rate_interval?

Reply:
@fasaxc Leon says: " I’m not 100% on what rate_interval is myself, I believe it’s what grafana believes the rate at which prometheus scrapes?"